outline.md

```diff
@@ -34,12 +36,14 @@ Ember-CLI provides choices for all of the aforementioned areas.  We'll dive into
     0. children in subdirectory → `routes/invoices/edit.js` & `routes/invoices/new.js`
 
 ### 5. Setup
+0. Instsall Git. This can be a bit different depending on your OS but a good resource is [git-scm](http://git-scm.com/downloads).
 0. Install Node.js.  Files and instructions can be found [here](https://nodejs.org/download/).
-0. Set up NPM for non-sudo installation
+0. Setup NPM for non-sudo installation
     0. NPM is the node package manager.  It will automatically be installed when you install node.
-    0. NPM installs packages *locally* (within the directory it is invoked in) for per-project modules, or *globally* for packages you want accessible everyewher.
+    0. NPM installs packages *locally* (within the directory it is invoked in) for per-project modules, or *globally* for packages you want accessible everyewhere.
```

Comment:
Fixing my typo, but there's still one in there... everyewhere should be everywhere